docs/sdk-and-tools/erdjs/migration-guides.md

```diff
@@ -9,9 +9,156 @@ This tutorial will guide you through the process of migrating from one major ver
+
+```
+    /**
+     * Creates a TransactionVersion object with the VERSION setting for enabling options
+     */
+    static withTxOptions(): TransactionVersion {
+        return new TransactionVersion(TRANSACTION_VERSION_WITH_TX_OPTIONS);
+    }
+```
+
+Also changes on [elrond-sdk-erdjs-walletcore](https://github.com/ElrondNetwork/elrond-sdk-erdjs-walletcore) have been implemented and will be made available with **v3.0.1** release (see [details](https://docs.elrond.com/sdk-and-tools/erdjs/erdjs-migration-guides#elrond-sdk-erdjs-walletcore---v301-release) below).
+
+### Guarded Transaction
+
+After setting a guardian (`SetGuardian`) and sending `GuardAccount` builtin transaction to Elrond network, only guardian co-signed transactions will be accepted by the *guarded* account. For supporting this, two new fields have been added to the transaction proto (Guardian Address and Guardian Signature):
+```
+message Transaction {
+	uint64   Nonce          = 1  [(gogoproto.jsontag) = "nonce"];
+	bytes    Value          = 2  [(gogoproto.jsontag) = "value", (gogoproto.casttypewith) = "math/big.Int;github.com/ElrondNetwork/elrond-go/data.BigIntCaster"];
+	bytes    RcvAddr        = 3  [(gogoproto.jsontag) = "receiver"];
+	bytes    RcvUserName    = 4  [(gogoproto.jsontag) = "rcvUserName,omitempty"];
+	bytes    SndAddr        = 5  [(gogoproto.jsontag) = "sender"];
+	bytes    SndUserName    = 6  [(gogoproto.jsontag) = "sndUserName,omitempty"];
+	uint64   GasPrice       = 7  [(gogoproto.jsontag) = "gasPrice,omitempty"];
+	uint64   GasLimit       = 8  [(gogoproto.jsontag) = "gasLimit,omitempty"];
+	bytes    Data           = 9  [(gogoproto.jsontag) = "data,omitempty"];
+	bytes    ChainID        = 10 [(gogoproto.jsontag) = "chainID"];
+	uint32   Version        = 11 [(gogoproto.jsontag) = "version"];
+	bytes    Signature      = 12 [(gogoproto.jsontag) = "signature,omitempty"];
+	uint32   Options        = 13 [(gogoproto.jsontag) = "options,omitempty"];
+	bytes    GuardAddr      = 14 [(gogoproto.jsontag) = "guardian,omitempty"];
+	bytes    GuardSignature = 15 [(gogoproto.jsontag) = "guardianSignature,omitempty"];
+}
+```
+Once an account is guarded, only transactions with set `guardian` and `version=2`, `options=2` will be accepted.
+
+Example:
+```
+const transaction = new Transaction({
+    nonce: sender.account.getNonceThenIncrement(),
+    sender: sender.address,
+    receiver: receiver.address,
+    value: payment,
+    gasLimit: gasLimit,
+    chainID: this.networkConfig.ChainID,
+    guardian: guardian.address,
+    options: TransactionOptions.withTxGuardedOptions(),
+    version: TransactionVersion.withTxOptions(),
+});
+```
+After adding sender's signature also guardians signature has to be added:
+```
+const guardian: IGuardianUser;
+...
+await guardian.signer.guard(transaction);
+```
+
+### Transaction field `sender` became mandatory with erdjs - v11
```

Comment:
Perhaps also add a hint to the month (August / September 2022)?

Reply:
Or keep this as it is, but add an extra section for `## Migrate **erdjs** from v10.x to v11 (summer of 2022)`?